### PR TITLE
DeversiFi: Add DVF token

### PR DESCRIPTION
**Pull request by plutoegg**

### CODE REVIEW

File: config/src/tokens.ts [hunk 1]
```diff
@@ -424,6 +424,13 @@ export const tokenList: TokenInfo[] = [
     decimals: 18,
     sinceBlock: 11305469,
   },
+  {
+    name: 'DVF',
```
Comment (sz-piotr) [inline config/src/tokens.ts:428]: Rename to "DeversiFi Token" because that is the name used in the smart contract

**Review by sz-piotr — CHANGES_REQUESTED** [review]

File: config/src/projects/deversifi.ts [hunk 2]
```diff
@@ -44,6 +44,7 @@ export const deversifi: Project = {
         'LRC',
         'OMG',
         'BAT',
+        'DVF',
```
Comment (sz-piotr) [inline config/src/projects/deversifi.ts:47]: Please also add the `associatedToken: 'DVF',` property to the project definition. Place it between `bridges` and `details`

**Review by sz-piotr — APPROVED** [review]

**Comment by sz-piotr** [comment]
@plutoegg Thanks for the PR!

**Comment by plutoegg** [comment]
> @plutoegg Thanks for the PR!

NP, love what you guys are doing!

**Pull request merged**